src/components/TODOList/AddNewTODO.js

```diff
@@ -1,127 +1,130 @@
-import React, { useState } from 'react';
-import {useDispatch, useSelector} from "react-redux";
-import DatePicker from "react-datepicker";
-import {Button, Card, Dialog, Input, Textarea} from "@material-tailwind/react";
-import {addTODO} from "../../store/TODOList_Slice";
+import React, {useState} from 'react';
```

Comment:
Is your linter not turned on?

Reply:
I turned it on I think

Reply:
Ok I will just add it in the repo in the next merge so its always on.

Reply:
Thank you very much. It seems that prettier does not work automatically in webstorm. I think I have a click "reformat code" explicitly to trigger it? I guess in something like Vscode it runs instantly?